app/controllers/teams_controller.rb

```diff
@@ -26,11 +26,11 @@ def create
     if current_user && current_user.admin?
     else
       flash[:alert] = "You do not have permission to create teams."
-      if team.save
-        redirect_to team
-      else
-        render :new, locals: { team: team }
-      end
+    end
+    if team.save
+      redirect_to team
+    else
+      render :new, locals: { team: team }
```

Comment:
I'm not sure I understand what's going on here. I don't think it was working before either though - I think lines 30-33 should be between line 26 & 27.